go/enclave/enclave.go

```diff
@@ -212,6 +212,11 @@ func NewEnclave(
 	}
 }
 
+func (e *enclaveImpl) GetID() (common.EnclaveID, error) {
+	// TODO: return real ID
+	return nil, nil
```

Comment:
I'd say to return the `e.config.HostId` - converted to EnclaveID.
And leave a todo